src/components/pager/pager.jsx

```diff
@@ -2,66 +2,44 @@ import React, { useEffect } from "react";
 import { Link } from "gatsby";
 
 import PropTypes from "prop-types";
-
-import { Pages } from "../pages";
+import { actualCategory } from "../../helpers/actual-category";
 
 import * as styles from "./pager.module.scss";
 
-
-const setPrevPage = ({currentPage}) => {
+const setPrevPage = ({currentPage, category}) => {
   if (currentPage === 1) {
     return 'javascript:void(0)'
   }
   
-  if ((currentPage - 1) === 1) {
+  if (!category && (currentPage - 1) === 1) {
     return '/'
   }
 
-  return `/page/${(currentPage - 1)}`
+  if (category && (currentPage - 1) === 1) {
+    return `/categories/${category}`;
+  }
+
+  return actualCategory() ? `/categories/${actualCategory()}/page/${(currentPage - 1)}` : `/page/${(currentPage - 1)}`;
```

Comment:
what's the difference between `actualCategory()` and param category?

Reply:
They are the same
Done